component-samples/map-components/templates/cdn/index.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html lang="en" dir="ltr">
```

Comment:
@jimmychen97 for consistency, unless `lang` and `dir` is required for charts components we should remove them from these samples to try to keep them as simple as possible.